### DISCUSSION TITLE
Unit 5 is live!

### DISCUSSION THREAD

**Discussion by rasbt** [Lightning-AI/dl-fundamentals#23] [Announcements]
Great news: Unit 5 is finally live! In Unit 5, I'll show you how to train PyTorch models with the Lightning Trainer!

https://lightning.ai/pages/courses/deep-learning-fundamentals/overview-organizing-your-code-with-pytorch-lightning/

**Comment by cornuch** [comment]
Shall we use pip install lightning or pip install pytorch-lightning  ?

**Reply by rasbt** [reply]
I recommend using `pip install lightning` (`pip install pytorch-lightning` is an older/legacy way to import it)

**Comment by cornuch** [comment]
Thank you for your quick reply. 

best regards 

CC 



De: "Sebastian Raschka" ***@***.***> 
À: "Lightning-AI/dl-fundamentals" ***@***.***> 
Cc: "CHRISTELLE CORNU" ***@***.***>, "Comment" ***@***.***> 
Envoyé: Lundi 1 Mai 2023 17:57:05 
Objet: Re: [Lightning-AI/dl-fundamentals] Unit 5 is live! (Discussion #23) 





I recommend using pip install lightning ( pip install pytorch-lightning is an older/legacy way to import it)
